src/quo/components/selectors/react/view.cljs

```diff
@@ -25,4 +25,4 @@
       {:on-press            on-press-add
        :state               :add-reaction
        :use-case            use-case
-       :accessibility-label (str "emoji-add-reaction")}])])
```

Comment:
Really weird we have this kind of code 🤔

Reply:
Also was surprised, but yeah, probably a leftover of some refactorings